acceptance/config/beaker/jenkins/redhat7-64m-64a.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
File extension shouldn't matter but I'll change it to *.cfg for consistency.  I'd like to leave the leading `---` as that's part of the YAML spec.